runtime/j9vm/java11vmi.c

```diff
@@ -1812,7 +1812,24 @@ JVM_GetNestMembers(JNIEnv *env, jclass clz)
 JNIEXPORT jboolean JNICALL
 JVM_AreNestMates(JNIEnv *env, jclass clzOne, jclass clzTwo)
```

Comment:
When the stub method was introduced, it wasn't immediately clear what these two `class` variables mean. Now it could be changed to `nestOne` & `nestTwo` or something else more meaningful.

Reply:
classOne and classTwo still makes the most sense as that's what we are comparing. We are trying to see if two classes belong to the same nest